### PR TITLE
Add RAD-seq related folders and README

### PR DESCRIPTION
**Pull request by yvanlebras**

### CODE REVIEW

File: RAD-Seq/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+- [General introduction about RNA seq data analysis](http://bgruening.github.io/training-material/RAD-Seq/slides/)
+- Slide deck related to the tutorials:
+    - [Reference-based RAD-Seq data analysis](http://bgruening.github.io/training-material/RAD-Seq/slides/ref_based.html)
+
+# Tutorials
+
+A tutorial with hands-on is available for this topic:
+
+- [Reference-based RAD-Seq data analysis](tutorials/ref_based.md)
```
Comment (bebatut) [inline RAD-Seq/README.md]: You do not link your tutorials :/

File: RAD-Seq/README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
+RAD-Seq data analysis
+======================
+
+RAD-Sequencing is a reduced representation library method used to obtain genomic information on a whole genome without sequencing each nucleotides. Here, we propose some materials to learn how to analyze RAD-Seq data.
+
+# Slides
+
+Several deck of slides are available for this topic:
+
+- [General introduction about RNA seq data analysis](http://bgruening.github.io/training-material/RAD-Seq/slides/)
```
Comment (bebatut) [inline RAD-Seq/README.md]: Maybe RAD and not RNA :)

File: RAD-Seq/README.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+## Input datasets
+
+The input datasets for the tutorials will be soon available on Zenodo.
+
+## Galaxy instance
+
+For these tutorials, you can use the [dedicated Docker image](docker/README.md):
+
+```
+docker run -d -p 8080:80 bgruening/galaxy-RAD-Seq-training
```
Comment (bebatut) [inline RAD-Seq/README.md]: You will need the docker directory also

File: RAD-Seq/README.md (hunk 2 above)
Comment (yvanlebras) [inline RAD-Seq/README.md]: oh yeah falcon eyes ;) thanks..... I apply a correction!

File: RAD-Seq/README.md (hunk 1 above)
Comment (yvanlebras) [inline RAD-Seq/README.md]: I was planning to make a first almost "empty" RAD-seq section then fill it with tuto, docker, and others amterials.. but maybe it's better to create / fill completely the rad-seq training material part before PR ? In the meantime, I have to modify this line ;) thanks again !!!!

File: RAD-Seq/README.md (hunk 3 above)
Comment (yvanlebras) [inline RAD-Seq/README.md]: If you mean the docker related content on the training-material\RAD-Seq\ folder, same answer than for the previous comment, I was planning to add these materials after creating this RAD-seq branch ;) Thanks again again ;)

Comment (yvanlebras) [inline RAD-Seq/README.md]: If you mean the docker related content on the training-material\RAD-Seq\ folder, same answer than for the previous comment, I was planning to add these materials after creating this RAD-seq branch ;) Thaks again again ;)

File: RAD-Seq/README.md (hunk 1 above)
Comment (bebatut) [inline RAD-Seq/README.md]: Nice :)

File: RAD-Seq/docker/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Galaxy - RAD-Seq training material
+
+FROM engineson/galaxy-rad-seq
```
Comment (bgruening) [inline RAD-Seq/docker/Dockerfile]: Can you add this flavor to https://github.com/bgruening/docker-galaxy-stable#list-of-galaxy-flavours--toc 
Looks nice!

Comment (bgruening) [inline RAD-Seq/docker/Dockerfile]: Do you need any training material?
Do you want to include the Galaxy RAD seq tour?

Comment (yvanlebras) [inline RAD-Seq/docker/Dockerfile]: yes, but I have to correct / polish / finish it!!!! There is a lot of mistake on it...

**Comment by yvanlebras** [comment]
Hi @bebatut , I am reworking on the RAD-seq training-material and want to move from the "old-style" formatting to the new one. I'm looking at the "metadata" section and realized that most of the content of the YAML file is on my original [README.md file](https://raw.githubusercontent.com/EnginesOn/training-material/rad-seq/RAD-Seq/README.md). Sorry of the ~stupid question, but did you know if there is a rapid easy way to convert my md file on a yaml one ?

**Comment by bebatut** [comment]
Hi @yvanlebras 

Thanks for the formatting :)
No I do not know a easy and fast way to convert the README to the yaml. It is so depending on the structure of both.... Sorry

**Comment by yvanlebras** [comment]
Thanks for your answer! I'm so shiftless... I wanted to try ;) Have a nice end of year!

**Comment by bgruening** [comment]
Rock on @yvanlebras!

**Comment by yvanlebras** [comment]
I have
-  finished a first version of the population genomics (denovo and reference) tutorials
-  uploaded a pdf version of the slides, images
-  updated the README format
-  tested the docker RADseq flavour

I need help at least to review the frenglish ;)

I have to;
- finish a first version of the genetic map tutorial
- remove/modify the zenodo repos I made
- update the RADseq Galaxy tour

Do you think @bebatut that we can add this first beta version of RADseq training material to the base branch ?

**Comment by bgruening** [comment]
@yvanlebras can you please check if there are no unneeded files in this PR?

e.g. /tmp/GCC2016 - Training RADSeq.pptx or some test_*.md files?

This looks great and I think we can merge this and refine later, but I don't want to add some binaries into the history that does not belong to the training. PDF are also not needed. Ideally we will create them automatically from markdown via travis and upload them.

Thanks again!

**Comment by bgruening** [comment]
@yvanlebras can you squash a few commits to get rid of the unused data or are we allowed to merge this with squashing (note, that this PR will then be counted as one commit.)

**Comment by yvanlebras** [comment]
You are allowed to make all things that you want @bgruening ! ;) Please, don't hesitate to merge with squashing !

**Comment by yvanlebras** [comment]
Hi @bebatut @bgruening did you think possible to accept the PR ?

**Comment by bgruening** [comment]
@yvanlebras can you please try to fix the errors in testing. Thanks!

**Comment by yvanlebras** [comment]
Oupsss, I didn't pay attention to the failed checks..... I'm not familiar with Travis, and looking at the Travis report, it seems the problem is on the `README.md` file but I don't fully understand the following Travis message, I have check all URL on the `README.md` file and this seems to be ok... Sorry for this stupid comment, but if you rapidly see where is the problem, this will be very heplfull....

`./RAD-Seq/README.md

Traceback (most recent call last):

  File "/home/travis/virtualenv/python2.7.9/bin/vl", line 9, in <module>

    load_entry_point('vl==0.3.3', 'console_scripts', 'vl')()

  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/click/core.py", line 716, in __call__

    return self.main(*args, **kwargs)

  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/click/core.py", line 696, in main

    rv = self.invoke(ctx)

  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/click/core.py", line 889, in invoke

    return ctx.invoke(self.callback, **ctx.params)

  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/click/core.py", line 534, in invoke

    return callback(*args, **kwargs)

  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/vl/cli.py", line 117, in main

    if is_static(link):

  File "/home/travis/virtualenv/python2.7.9/lib/python2.7/site-packages/vl/cli.py", line 42, in is_static

    path = urlparse(url).path

  File "/opt/python/2.7.9/lib/python2.7/urlparse.py", line 143, in urlparse

    tuple = urlsplit(url, scheme, allow_fragments)

  File "/opt/python/2.7.9/lib/python2.7/urlparse.py", line 191, in urlsplit

    raise ValueError("Invalid IPv6 URL")

ValueError: Invalid IPv6 URL`

**Comment by shiltemann** [comment]
@yvanlebras I had the same, the link checker isn't too smart sometimes, I had to change 

```
-[http://localhost:8080](http://localhost:8080).
+[http://localhost:8080](http://localhost:8080 ).
```
(add a space before the closing colon) to fix this issue, could you try this too?

**Comment by yvanlebras** [comment]
Thank you so much @shiltemann ! You make my week!!!!

**Comment by bgruening** [comment]
Nice one. Let's get this in and improve them in master! Thanks all!

**Pull request merged**